integration/fixtures/server-data/access.txt

```diff
@@ -0,0 +1 @@
+TestUser	test@user.com	0000-0000-1234-0000	TestOrganization	d72936dcb7db05c6d890df30ea173a38	0
```

Comment:
Probably better to use `testuser@example.com`. `Example.com` is reserved for stuff like this.